timeless/views.py

```diff
@@ -58,6 +58,25 @@ class CrudAPIView(views.MethodView):
      result.  Please refer to #221 and #222 for documentation.
     """
 
+    @classmethod
+    def register(cls, blueprint, route, name=None):
+        """
+        A shortcut method for registering this view to an app or blueprint.
+        Assuming we have a blueprint and a CompanyCreate view, then these two
+        lines are identical in functionality:
+            views.add_url_rule('/companies/create',
+                               view_func=CompanyCreate.as_view(
+                                   'company_create')
+                               )
+            CompanyCreate.register(views, '/companies/create',
+                                   'company_create')
+        """
+        if not name:
+            # Convert "ViewName" to "view_name" and use it
+            name = camel_to_underscore.sub(r"_\1", cls.__name__).lower()
```

Comment:
@emilianodellacasa can't we add this ``camel_to_...`` as default value in the function's signature (instead of ``None``)?

Reply:
@amihaiemil the default value comes into play when parameter is not pased to the function, in this case we are just transforming a parameter

Reply:
@emilianodellacasa aha, the same code is in `GenericView`, I guess we can create a base class (or mixin) with this method :)